### DISCUSSION TITLE
Self-hosted runners removal

### DISCUSSION THREAD

**Discussion by Asiberus** [community/community#160556] [Actions]
Hello, 

I was wondering why do a self-hosted runners remove itself from the runners list after 30 days. 
I work on a repo with a deployment action that is used approximatively every 3/4 month and I have to reconfigure the action runner each time. 
I can understand this limitation for non self-hosted runners, but for self-hosted configured by the users they should always be up and waiting for jobs in queue.
Plus, the reconfiguration of a removed runners is really not straighforward. We need a `token` to use the `config.sh remove` script, but as the runners is removed on the github repository we have lost the runner's token.

**Comment by aveXcaesar** [comment]
You can skip the server site removal by deleting (or renaming) the `.credentials` file in the runner root directory. After that the remove script skips the server site removal.
See my output:

```cmd
# Runner removal

Cannot connect to server, because config files are missing. Skipping removing runner from the server.
Does not exist. Skipping Removing .credentials
√ Removed .runner
```

**Reply by mkaczmarek-evenica** [reply]
Thank you. This is a nonsense issue that GitHub created and refuses to address, I'm glad at least there is a simple workaround. Now if only GitHub would at least tell people how to do this...
